public/locales/sv/common.json

```diff
@@ -1,8 +1,10 @@
 {
   "appName": "Linked Registrations",
+  "ariaLogo": "Service logo",
```

Comment:
I think "Tjänstens logo" is the proper Swedish translation

Reply:
Changed